bin/analyze_hoc_activity

```diff
@@ -101,15 +101,21 @@ def main()
     # Since s3 (which the binary relies on) may fail, we retry a few times
     # hoping for success.
     3.times do
-      # Note that by executing via backticks, the global variable $? is set.
+      # Note that by executing via backticks, the global variable $? is set to a Process::Status object.
       lines_of_code_from_milestone = `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i
```

Comment:
The milestone_parser.rb seems to have some (sufficient) debugging already, via a debug(msg) method defined therein, gated on a cattr_accessor variable log_debug (used in the test). Presumably, to enable it in prod, I need to set log_debug to true? Doing so as part of this PR. Where do the messages get output?